src/components/getHelpModal/GetHelpModal.js

```diff
@@ -1,69 +1,55 @@
-import { Button, Header, Icon, Modal, Table } from "semantic-ui-react";
-import React, { Component } from "react";
```

Comment:
-  functional component still need to add  `import React from 'react' `

Reply:
They don’t need to anymore since React 17:
https://stackoverflow.com/questions/44141010/do-i-need-to-import-react-for-stateless-functional-components

Reply:
ahhh right this component is stateless and there were some errors when I used React 16.

Good to know! I've tested it again with React 17 and everything works!  😃😃